### PR TITLE
Deprecated syntax removal for variable declaration

### PR DESCRIPTION
**Pull request by saurabh896**
Hope this will fix the error.
We need to test echo command though.

### CODE REVIEW

**Comment by saurabh896** [comment]
@ErikSchierboom  I think this will fix variable declaration error.

File: .github/workflows/dockerhub.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Set exercism config version
-        run: echo ::set-env name=EXERCISM_CONFIG_VERSION::$(grep 'exercism-config (' Gemfile.lock | grep -o '[0-9.]\+')
+        run: echo "EXERCISM_CONFIG_VERSION::$(grep 'exercism-config (' Gemfile.lock | grep -o '[0-9.]\+')" >> $GITHUB_ENV
```
Comment (ErikSchierboom) [inline .github/workflows/dockerhub.yml:16]: @saurabh896 I think the syntax also has to be updated (see [the docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable)):

```suggestion
        run: echo "EXERCISM_CONFIG_VERSION=$(grep 'exercism-config (' Gemfile.lock | grep -o '[0-9.]\+')" >> $GITHUB_ENV
```

Right @SaschaMann?

**Review by ErikSchierboom — CHANGES_REQUESTED** [review]

File: .github/workflows/dockerhub.yml (hunk 1 above)
Comment (saurabh896) [reply .github/workflows/dockerhub.yml]: Oh!! I missed this one. thanks and I fixed echo too. env.variable_name is deprecated too.

**Review by ErikSchierboom — APPROVED** [review]
LGTM!

**Comment by ErikSchierboom** [comment]
@saurabh896 This does indeed fix #52! See [the logs](https://github.com/exercism/development-environment/runs/1478679434?check_suite_focus=true). Thanks a lot!

**Comment by saurabh896** [comment]
@ErikSchierboom  If you have any CI issue let me know I can look into that..

**Comment by ErikSchierboom** [comment]
Great, thanks!

**Pull request merged**